src/main.c

```diff
@@ -100,6 +101,13 @@ int main(int argc, char** argv)
         return -1;
     }
 
+    registerReturn = UT_register_APIDEF_l3_tests();
```

Comment:
please re-base branch, registration is already done

Reply:
Rebased and updated.